File: send_email.py
import smtplib
from email.mime.text import MIMEText
from email.mime.multipart import MIMEMultipart
from datetime import datetime

def send_email(gmail_user, gmail_password, to_email):
    try:
        # Email content
        subject = "Automated Notification"
        body = f"This is an automated notification sent at {datetime.now()}."

        # Create the email
        msg = MIMEMultipart()
        msg["From"] = gmail_user
        msg["To"] = to_email
        msg["Subject"] = subject
        msg.attach(MIMEText(body, "plain"))

        # Connect to Gmail server and send email
        with smtplib.SMTP("smtp.gmail.com", 587) as server:
            server.starttls()  # Secure the connection
            server.login(gmail_user, gmail_password)
            server.send_message(msg)

        print(f"Email sent successfully at {datetime.now()}")
    except Exception as e:
        print(f"Failed to send email: {e}")

if __name__ == "__main__":
    import os
    gmail_user = os.getenv("GMAIL_USER")
    gmail_password = os.getenv("GMAIL_PASSWORD")
    to_email = os.getenv("TO_EMAIL")
    send_email(gmail_user, gmail_password, to_email)

File: send_email.php
<?php
// Include PHPMailer library
require 'PHPMailer/PHPMailer.php';
require 'PHPMailer/SMTP.php';
require 'PHPMailer/Exception.php';

use PHPMailer\PHPMailer\PHPMailer;
use PHPMailer\PHPMailer\Exception;

header('Content-Type: application/json');

// Configure email parameters
$mail = new PHPMailer(true);
try {
    // Server settings
    $mail->isSMTP();
    $mail->Host = 'smtp.gmail.com'; // Gmail SMTP server
    $mail->SMTPAuth = true;
    $mail->Username = '@gmail.com'; // Your Gmail address
    $mail->Password = 'your-email-password'; // Your Gmail password or App Password
    $mail->SMTPSecure = PHPMailer::ENCRYPTION_STARTTLS;
    $mail->Port = 587;

    // Recipient and sender
    $mail->setFrom('your-email@gmail.com', 'Your Name');
    $mail->addAddress('recipient-email@gmail.com', 'Recipient Name'); // Recipient email and name

    // Content
    $mail->isHTML(true);
    $mail->Subject = 'Automated Email on Page Reload';
    $mail->Body = '<h1>Hi!</h1><p>This is an automated email sent when the page was reloaded.</p>';

    // Send the email
    $mail->send();
    echo json_encode(['status' => 'success', 'message' => 'Email sent successfully!']);
} catch (Exception $e) {
    echo json_encode(['status' => 'error', 'message' => "Email could not be sent. Error: {$mail->ErrorInfo}"]);
}
